src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -574,6 +576,10 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
       logger.debug(s"Got whenUnhandled PersistenceCompleted(${callKey.tag}): $status")
       val updatedData = data.removePersisting(callKey, status)
       stay using updatedData
+    case Event(PersistenceFailed(callKey, status), data) =>
+      logger.debug(s"Got whenUnhandled PersistenceFailed(${callKey.tag}): $status")
+      val updatedData = data.removePersisting(callKey, status)
+      stay using updatedData
```

Comment:
this and the above could probably be pulled out to a function using the default case class toString for the message